[2016-11-08 14:58:25] DEBUG [django.db.backends:90] (0.004) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
[2016-11-08 14:58:25] DEBUG [django.db.backends:90] (0.000) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
[2016-11-08 14:59:50] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-11-08 14:59:50] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-11-08 14:59:58] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_application" WHERE "oauth2_provider_application"."client_id" = 'Nd33HkotycCIWVTPQEuLPz7VyAw05ioaGJJy0gJb'; args=('Nd33HkotycCIWVTPQEuLPz7VyAw05ioaGJJy0gJb',)
[2016-11-08 14:59:58] DEBUG [django.db.backends:90] (0.001) SELECT "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by" FROM "auths_user" WHERE "auths_user"."username" = 'admin'; args=('admin',)
[2016-11-08 14:59:59] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-08 14:59:59] DEBUG [django.db.backends:90] (0.061) INSERT INTO "oauth2_provider_accesstoken" ("user_id", "token", "application_id", "expires", "scope") VALUES ('25e7efce92ee410c84e90b3c8ba9e969', 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy', 1, '2016-11-09 05:59:59.000189', 'write read'); args=['25e7efce92ee410c84e90b3c8ba9e969', 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy', 1, '2016-11-09 05:59:59.000189', 'write read']
[2016-11-08 14:59:59] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-08 14:59:59] DEBUG [django.db.backends:90] (0.002) INSERT INTO "oauth2_provider_refreshtoken" ("user_id", "token", "application_id", "access_token_id") VALUES ('25e7efce92ee410c84e90b3c8ba9e969', '1gK7WxlZL6kEMfPrT5myypFCyRp5jh', 1, 3); args=['25e7efce92ee410c84e90b3c8ba9e969', '1gK7WxlZL6kEMfPrT5myypFCyRp5jh', 1, 3]
[2016-11-08 14:59:59] INFO [django.server:131] "POST /o/token/ HTTP/1.1" 200 169
[2016-11-08 15:00:03] INFO [django.server:131] "OPTIONS /api/auths/localuserinfo/ HTTP/1.1" 200 0
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.003) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy'; args=('GfcsTHsfDW7jgWzfCv9KNEUThTxBSy',)
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.004) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy'; args=('GfcsTHsfDW7jgWzfCv9KNEUThTxBSy',)
[2016-11-08 15:00:03] INFO [django.server:131] "OPTIONS /api/auths/usermenu/ HTTP/1.1" 200 0
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.001) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '25e7efce92ee410c84e90b3c8ba9e969'; args=('25e7efce92ee410c84e90b3c8ba9e969',)
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.006) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '25e7efce92ee410c84e90b3c8ba9e969' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('25e7efce92ee410c84e90b3c8ba9e969',)
[2016-11-08 15:00:03] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.004) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy'; args=('GfcsTHsfDW7jgWzfCv9KNEUThTxBSy',)
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy'; args=('GfcsTHsfDW7jgWzfCv9KNEUThTxBSy',)
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."is_active" = 1 AND "auths_menu"."module" = 'BACKEND') ORDER BY "auths_menu"."pos" ASC; args=(True, 'BACKEND')
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.000) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'wavivjo14z4uro87q6ffcl61667zkwf3' LIMIT 1; args=('wavivjo14z4uro87q6ffcl61667zkwf3',)
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-11-08 15:00:03] DEBUG [django.db.backends:90] (0.002) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'wavivjo14z4uro87q6ffcl61667zkwf3', 'Y2U1MDdhOTVhY2Y1M2EzOGYyOTc1YmE0OTQ4NTlmMWJkODNmMmEzMjp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-08 20:00:33.241114'; args=('wavivjo14z4uro87q6ffcl61667zkwf3', 'Y2U1MDdhOTVhY2Y1M2EzOGYyOTc1YmE0OTQ4NTlmMWJkODNmMmEzMjp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-11-08 20:00:33.241114')
[2016-11-08 15:00:03] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-11-08 15:00:12] INFO [django.server:131] "OPTIONS /api/catalogo/cliente/ HTTP/1.1" 200 0
[2016-11-08 15:00:12] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy'; args=('GfcsTHsfDW7jgWzfCv9KNEUThTxBSy',)
[2016-11-08 15:00:12] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'GfcsTHsfDW7jgWzfCv9KNEUThTxBSy'; args=('GfcsTHsfDW7jgWzfCv9KNEUThTxBSy',)
[2016-11-08 15:00:12] DEBUG [django.db.backends:90] (0.001) SELECT COUNT(*) AS "__count" FROM "catalogo_cliente"; args=()
[2016-11-08 15:00:12] DEBUG [django.db.backends:90] (0.001) SELECT "catalogo_cliente"."id", "catalogo_cliente"."nombre", "catalogo_cliente"."apellidos", "catalogo_cliente"."dni", "catalogo_cliente"."direccion", "catalogo_cliente"."fecha_nac", "catalogo_cliente"."created_at", "catalogo_cliente"."updated_at" FROM "catalogo_cliente" LIMIT 1; args=()
[2016-11-08 15:00:12] ERROR [django.request:124] Internal Server Error: /api/catalogo/cliente/
Traceback (most recent call last):
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\django\views\decorators\csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\viewsets.py", line 83, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\views.py", line 477, in dispatch
    response = self.handle_exception(exc)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\views.py", line 437, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\views.py", line 474, in dispatch
    response = handler(request, *args, **kwargs)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\mixins.py", line 45, in list
    return self.get_paginated_response(serializer.data)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 725, in data
    ret = super(ListSerializer, self).data
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 643, in to_representation
    self.child.to_representation(item) for item in iterable
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 643, in <listcomp>
    self.child.to_representation(item) for item in iterable
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 479, in to_representation
    fields = self._readable_fields
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\django\utils\functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 373, in _readable_fields
    field for field in self.fields.values()
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 359, in fields
    for key, value in self.get_fields().items():
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 980, in get_fields
    field_names = self.get_field_names(declared_fields, info)
  File "D:\pydev\lp\marketsoft\ve_market\lib\site-packages\rest_framework\serializers.py", line 1053, in get_field_names
    serializer_class=self.__class__.__name__
AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the ClienteSerializer serializer.",)
[2016-11-08 15:00:12] DEBUG [django.db.backends:90] (0.001) SELECT "catalogo_cliente"."id", "catalogo_cliente"."nombre", "catalogo_cliente"."apellidos", "catalogo_cliente"."dni", "catalogo_cliente"."direccion", "catalogo_cliente"."fecha_nac", "catalogo_cliente"."created_at", "catalogo_cliente"."updated_at" FROM "catalogo_cliente" LIMIT 21; args=()
[2016-11-08 15:00:12] ERROR [django.server:131] "GET /api/catalogo/cliente/ HTTP/1.1" 500 20519
